cluster-scope/overlays/nerc-ocp-prod/metallb/kustomization.yaml

```diff
@@ -2,6 +2,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 namespace: metallb-system
 resources:
-- ipaddresspools/public.yaml
-- l2advertisements/default.yaml
```

Comment:
do you want to remove this file too?

Reply:
I was thinking of keeping them for when we upgrade to 4.11 at some point, but I can go ahead and remove them.

Reply:
Deleted.